src/main/java/dev/jbang/source/Project.java

```diff
@@ -164,6 +164,31 @@ public Project setJavaVersion(String javaVersion) {
 		return this;
 	}
 
+	@Nonnull
+	public String getFullJavaVersion() {
+		String fullJavaVersion = getJavaVersion();
+		if (fullJavaVersion == null) {
+			// Get Java version from running JVM
+			fullJavaVersion = System.getProperty("java.version");
+		} else {
+			// The latest stable Java versions supported by SDKMAN.
+			// This map should be updated at least once a year
+			// Run command: $ sdk list java | grep tem
+			// to get a list of supported Temurin versions
+			Map<String, String> jv = new HashMap<String, String>();
+			jv.put("23", "23.0.2");
```

Comment:
also - this is 100% sdkman dependent - its not something to put into core Project.java as nothing else but the export uses it.

Reply:
@maxandersen , I agree. 

As a temporary measure, I'll just copy the .sdkmanrc file into the destination Maven or Gradle folder.